sos/report/plugins/pulpcore.py

```diff
@@ -24,16 +24,17 @@ class PulpCore(Plugin, IndependentPlugin):
         PluginOpt('task-days', default=7, desc='days of task history')
     ]
 
+    dbhost = "localhost"
+    dbport = 5432
+    dbname = "pulpcore"
+    dbpasswd = ""
+    staticroot = "/var/lib/pulp/assets"
+    uploaddir = "/var/lib/pulp/media/upload"
+    env = {"PGPASSWORD": dbpasswd}
+
     def parse_settings_config(self):
+        """ Parse pulp settings """
         databases_scope = False
-        self.dbhost = "localhost"
-        self.dbport = 5432
-        self.dbname = "pulpcore"
-        self.dbpasswd = ""
-        # TODO: read also redis config (we dont expect much customisations)
```

Comment:
This is relatively new - from 2021. Please let me know if this needs to tracked in an #issue.

Reply:
No need for a TODO. Afaik `pulp-3` aka `pulpcore` is not using now `redis`.